pyproject.toml

```diff
@@ -65,7 +65,7 @@ lgbm = ["lightgbm"]
 huggingface = ["transformers", "datasets"]
 catalyst = ["catalyst>22"]
 fastai = ["fastai"]
-lightning = ["lightning>=2.0", "torch"]
+lightning = ["lightning>=2.0", "torch", "jsonargparse[signatures]>=4.26.1"]
```

Comment:
Added to fix test failure in https://github.com/iterative/dvclive/actions/runs/7302568222/job/19901439469#step:10:502